pkg/tfshim/shim.go

```diff
@@ -214,16 +214,22 @@ type Provider interface {
 	InitLogging(ctx context.Context)
 
 	// Create a Destroy diff for a resource identified by the TF token t.
-	NewDestroyDiff(ctx context.Context, t string) InstanceDiff
+	NewDestroyDiff(ctx context.Context, t string, opts TimeoutOptions) InstanceDiff
```

Comment:
Specifically destroy diffs seem to be interested in the DELETE timeout handling.